### PR TITLE
Tweak: Docs - Container element [ED-3809]

### PR DESCRIPTION
**Pull request by StyleShit**

### CODE REVIEW

**Review by Nevoss — APPROVED** [review]

File: docs/core/container-element.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+- [Flex Cheatsheet](https://yoksel.github.io/flex-cheatsheet/)
+
+
+
+## Attention Needed / Known Issues:
+
+Creating a new core element in a legacy code-base is never easy. Therefore, we had to deal with many weird problems.
+
+The main one is the DnD aspect. DnD-ing into a `flex container` is a real challenge by itself, and when you add the fact that you need to support the old existing DnD mechanism, it becomes a huge pain in the a**.
```
Comment (matipojo) [inline docs/core/container-element.md:26]: Language!

Comment (matipojo) [inline docs/core/container-element.md:26]:
```suggestion
The main one is the DnD (Drag & Drop) aspect. DnD-ing into a `flex container` is a real challenge by itself, and when you add the fact that you need to support the old existing DnD mechanism, it becomes a huge pain in the a**.
```

File: docs/core/container-element.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Some things that we needed to take into consideration were:
+
+- What are you gonna do when the `flex-direction` is `row`?
+- How jQuery UI Draggable & Sortable are gonna treat dragging into a `flex container`?
+- How it will affect the existing placeholder mechanism that knows only about things like `top` / `bottom` / `vertical`?
+- How does it affect the UX/UI?
+- Will it cause issues due to `flex-basis` / `flex-shrink` / `flex-grow`?
+- How do you even resize a `flex item`?
```
Comment (matipojo) [inline docs/core/container-element.md:35]:
```suggestion
- How do you even resize a `flex item` if there are some calculations like grow/shrink?
```

**Review by matipojo — CHANGES_REQUESTED** [review]

File: docs/core/container-element.md (hunk 1 above)
Comment (StyleShit) [reply docs/core/container-element.md:26]: Done

Comment (StyleShit) [reply docs/core/container-element.md:26]: Done 😅

File: docs/core/container-element.md (hunk 2 above)
Comment (StyleShit) [reply docs/core/container-element.md:35]: Done.

Comment (StyleShit) [reply docs/core/container-element.md]: BTW, the file name/path are ok?

Comment (matipojo) [reply docs/core/container-element.md]: It should mirror the original file

**Review by matipojo — CHANGES_REQUESTED** [review]
Move to the `container` branch

**Pull request merged**